packages/esm-service-queues-app/translations/en.json

```diff
@@ -12,6 +12,7 @@
   "addPatientToQueue": "Add patient to queue",
   "addProviderQueueRoom": "Add provider queue room?",
   "addQueue": "Add queue",
+  "addQueueEntry": "Add queue entry",
```

Comment:
I would think this would have been removed after you run through the commit hook.  You might need to re-commit with the change to remove this.  @makombe